lib/components/camera.dart

```diff
@@ -210,11 +210,16 @@ class _CameraScreenState extends State<CameraScreen> {
   void onTakenPicture() {
     takePicture().then((String bigFilePath) async {
       final bigImage = Img.decodeImage(new File(bigFilePath).readAsBytesSync());
-      new File(bigFilePath)..writeAsBytesSync(Img.encodePng(bigImage));
-      if (mounted) {
-        setState(() {
-          bigImagePath = bigFilePath;
-        });
+      int ht = bigImage.height;
+      int wd = bigImage.width;
+      int reducedHt = (ht * 0.1).toInt();
+      int reducedWd = (wd * 0.1).toInt();
+      final croppedImage = Img.copyCrop(bigImage, reducedHt, reducedWd, wd, ht);
```

Comment:
To crop a rectangular image into a square, you have to first find out its dimensions - lets say imgWidth and imgHeight. Then figure out which is less - say imgHeight. Then set x = (imgWidth - imgHeight)/2. Set y = 0, width = imgHeight, height = imgHeight